src/main/scala/cognite/spark/v1/RawTableRelation.scala

```diff
@@ -34,6 +34,8 @@ class RawTableRelation(
   import CdpConnector._
   import RawTableRelation._
 
+  override def toString: String = s"RawTableRelation(db=${database}, table=${table})"
```

Comment:
Do we need the actual db & table names? Or is it enough to know that they are different db/tables?
My thinking is just the unintentional printing of info that shouldn't be printed